inc/hclib-async.h

```diff
@@ -188,6 +189,24 @@ inline void async_nb_await(T&& lambda, hclib_future_t *future) {
 	spawn_await(task, future ? &future : NULL, future ? 1 : 0);
 }
 
+template <typename T>
+inline void async_nb_await(T&& lambda, std::vector<hclib_future_t *> &futures) {
+	MARK_OVH(current_ws()->id);
```

Comment:
Looks like there's an alignment issue here. All lines should be indented by 4 spaces.

Reply:
Actually, looks like there's an alignment issue throughout this file. Since this is pre-existing, could you just file an issue saying that alignments are off in hclib-async.h to make sure I fix it?

Reply:
I will add an issue. It seems that this file used tab instead of space and based on local tab configuration it will show different different spacing.